src/site/content/en/blog/content-visibility/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 'CSS Content Visibility: the most impactful performance boost CSS can give you'
```

Comment:
@una are you sure about the statement that this is the most impactful performance boost? If not there are ways for us to rephrase this to avoid a false claim

Reply:
We are renaming it to: `content-visibility`: the new CSS property that boosts your rendering performance